webapp/packages/plugin-connections/src/ConnectionAuthService.ts

```diff
@@ -43,17 +43,12 @@ export class ConnectionAuthService extends Dependency {
   private async connectionDialog(connectionKey: IConnectionInfoParams, context: IExecutionContextProvider<IConnectionInfoParams | null>) {
     const connection = context.getContext(this.connectionsManagerService.connectionContext);
 
-    try {
-      const tempConnection = await this.auth(connectionKey);
+    const tempConnection = await this.auth(connectionKey);
 
-      if (!tempConnection?.connected) {
-        return;
-      }
-      connection.connection = tempConnection;
-    } catch (exception: any) {
-      this.notificationService.logException(exception);
-      throw exception;
+    if (!tempConnection?.connected) {
+      return;
     }
+    connection.connection = tempConnection;
```

Comment:
I removed logException here, because it is first logged here and then goes higher and is logged one more time (loadNestedNodes).

Reply:
nice